src/pages/AddRecord/AddRecordCategory.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import { RecordType } from './AddRecord'
+import Celebrate from '@assets/celebrate.svg'
+import Happy from '@assets/happy.svg'
+import Cake from '@assets/cake.svg'
+import Love from '@assets/love.svg'
+
+type userProps = {
+  currentRecordType: RecordType
+}
+
+function AddRecordCategory({ currentRecordType }: userProps) {
```

Comment:
RecordType이 그냥 string 타입인데 임포트를 하면서 하신 이유를 잘 모르겠네요
저는 속성이 하나밖에 없는 객체면 그냥 이런 식으로 줍니다

```suggestion
function AddRecordCategory({ currentRecordType }: {currentRecordType: string}) {
```

Reply:
굳이 string만 있다면 그럴 필요가 없겠네요 반영하겠습니다!